subprojects/docs/src/samples/spring-boot-web-application/groovy/app/build.gradle

```diff
@@ -23,6 +23,6 @@ dependencies {
 	}
 }
 
-tasks.named('test', Test) {
+tasks.withType(Test).configureEach {
```

Comment:
This is fine, as there is only one task of type `Test`, but note that the original only configured a task named 'test' of that type.